dot/state/service_test.go

```diff
@@ -33,12 +33,12 @@ import (
 // helper method to create and start test state service
 func newTestService(t *testing.T) (state *Service) {
 	testDir := utils.NewTestDir(t)
-	state = NewService(testDir)
+	state = NewService(testDir, 5)
```

Comment:
I wonder if we should express these calls as `state = NewService(testDir, log15.LvlTrace)` so that it's clear that the 5 represents log level Trace?  Here and other places, not sure it's "necessary" since these are in tests.

Reply:
that makes much more sense, I'll update it!